src/Microsoft.Framework.SecretManager/Program.cs

```diff
@@ -57,9 +58,13 @@ public int Main(string[] args)
             try
             {
                 var app = new CommandLineApplication();
+                app.FullName = "DNX UserSecrets";
                 app.Name = "user-secret";
                 app.Description = "Manages user secrets";
 
+                var assembly = typeof(Program).GetTypeInfo().Assembly;
+                app.ShortVersionGetter = () => assembly.GetName().Version.ToString();
```

Comment:
@kichalla Where are you seeing those attributes in the project? I don't seem to have those (text not found in entire solution).

@BrennanConroy I just tried the following:

    var attributes = assembly.GetCustomAttributes(
        typeof(AssemblyInformationalVersionAttribute)) 
        as AssemblyInformationalVersionAttribute[];

...but I only seem to get 1.0.0, without the beta tag and I suppose it's because I don't have the above attributes set?